Assets/Scripts/Client/Systems/UpdateMaterialComponentData.cs

```diff
@@ -11,31 +11,37 @@ namespace PropHunt.Client.Systems
     /// When a material of an entity needs to be updated, this system will update the render mesh.
     /// </summary>
     [UpdateInGroup(typeof(ClientSimulationSystemGroup))]
-    public class UpdateMaterialSystem : ComponentSystem
+    public class UpdateMaterialSystem : SystemBase
```

Comment:
Is there a reason this is switched to System Base? If it has to run without burst and can't be done in parallel it might be a better idea to leave it as a component system.

Reply:
Just wanted to "standarize" SystemBase, as it seems it's the most updated interface.
No other reason in particular. Kind of like an OCD thing of mine, but it can be reverted.

Reply:
Should be fine. We don't have an enforced style. We could change to use an entity buffer to improve the quality. Make sure it passes the unit tests (rebase with master to get the test). I will help if you need me to update the test to pass :)

Reply:
by entity buffer you mean entity command buffer? because that's what it's using